docs/frameworks/nextjs.md

```diff
@@ -0,0 +1,29 @@
+# Next.js
```

Comment:
While deploying NextJs on Genezio, I personally had the following questions that I think are worth answering in the documentation:

1. How can I manipulate the `subdomain` of my project? The answer is changing it in the `package.json` or in the `genezio.yaml`.
2. How can I check logs on the server side? 
3. How can I manipulate environment variables/secrets?
4. Can I define a custom domain?

There are also more advanced questions - let me know if you find it fit to add them to the docs also:

5. What are the `function-image-optimization` and `function-server`?
6. How can I test the API of my project/ How to use the Test Interface?
7. How do I test locally? 
8. Do I need a Docker/Dockerfile?
9.  `genezio deploy` creates the following artefacts - `.openext` and `genezio.yaml.` What is opennext? What configs from opennext should I push tp my repo? Should I push `.opennext` or should I add it to `.gitignore`?

Reply:
FYI, I have grouped the questions into 2 sets because I believe the second set can be added in a second iteration of this documentation page.